packages/mobile/src/identity/verification.ts

```diff
@@ -365,20 +366,20 @@ function attestationCodeReceiver(
         throw new Error('No code extracted from message')
       }
 
-      const issuer = findMatchingIssuer(e164NumberHash, account, code, allIssuers)
-      if (!issuer) {
-        throw new Error('No issuer found for attestion code')
-      }
-
-      const existingCode = yield call(getCodeForIssuer, issuer)
+      const existingCode = yield call(isCodeAlreadyAccepted, code)
       if (existingCode) {
         Logger.warn(TAG + '@attestationCodeReceiver', 'Code already exists in store, skipping.')
-        if (action.inputType === CodeInputType.MANUAL) {
+        if ([CodeInputType.MANUAL, CodeInputType.DEEP_LINK].indexOf(action.inputType) !== -1) {
```

Comment:
This seems like in inefficient way to do an `or` check ;)

Reply:
@jmrossy  I usually do this way, so when you add more stuff the `if` statement isn't growing to multiple `... === CodeInputType.MANUAL` over and over again. But this got me curious, how slow is it compare to `or` - will update here later with the results.

Reply:
It is indeed way slower (around 25-40 times on a high number of iterations). Will update it :) Thanks for pointing this out!